apps/schools/domains/user/components/auth/registerForm/index.tsx

```diff
@@ -28,12 +30,36 @@ export const RegisterForm: React.FC<IRegisterFormProps> = ({ onFinish, onError }
         signInByPhone: () => {},
     }
 
+    const [updateProfile] = useUpdateEmployeeProfileByIdMutation();
+    const { user } = useUserProfile()
```

Comment:
you can use lazy get user query instead of useUserProfile layout and use it after successful useUsersMutation